packages/amesos2/example/SimpleSolve_File.cpp

```diff
@@ -43,9 +43,11 @@ int main(int argc, char *argv[]) {
   typedef Tpetra::CrsMatrix<>::scalar_type Scalar;
   typedef Tpetra::Map<>::local_ordinal_type LO;
   typedef Tpetra::Map<>::global_ordinal_type GO;
-  typedef Tpetra::Map<>::node_type NO;
 
+#if defined(HAVE_AMESOS2_XPETRA) && defined(HAVE_AMESOS2_ZOLTAN2)
+  typedef Tpetra::Map<>::node_type NO;
   typedef Tpetra::RowGraph<LO, GO, NO> Graph;
```

Comment:
NO and Graph are only available as a type when `HAVE_AMESOS2_XPETRA` and `HAVE_AMESOS2_ZOLTAN2` are true, should there be a type for Graph without NO when one of the checks fail? Or was this to clean up an unused warning?